src/domain/community/organisation/organisation.resolver.mutations.ts

```diff
@@ -68,4 +68,15 @@ export class OrganisationResolverMutations {
     );
     return org;
   }
+
+  @Roles(AuthorizationRoles.EcoverseAdmins)
```

Comment:
Discussion: EcoverseAdmin should be able to delete only orgs in his eco verse context.

Reply:
Interesting one, and has the same implication for Users - it may be that Community Admin is not a subset of EcoverseAdmin. Authorisation in general needs a discussion re multi-ecoverse support.